Source/ZoomNet/Resources/Phone.cs

```diff
@@ -62,6 +63,37 @@ public Task<PhoneCallUserProfile> GetPhoneCallUserProfileAsync(string userId, Ca
 				.AsObject<PhoneCallUserProfile>();
 		}
 
+		/// <inheritdoc cref="IPhone.ListPhoneCallUserProfilesAsync" select="remarks" />
+		public Task<PhoneCallUserProfilesPaginationObject> ListPhoneCallUserProfilesAsync(
+			int pageSize = 30,
+			string nextPageToken = null,
+			string siteId = null,
+			int? callingType = null,
+			UserStatus? status = null,
+			string department = null,
+			string costCenter = null,
+			string keyword = null,
+			CancellationToken cancellationToken = default)
+		{
+			if (pageSize < 1 || pageSize > 100)
+			{
+				throw new ArgumentOutOfRangeException(nameof(pageSize), "Records per page must be between 1 and 100");
+			}
+
+			return _client
+				.GetAsync($"phone/users")
+				.WithArgument("page_size", pageSize)
+				.WithArgument("next_page_token", nextPageToken)
+				.WithArgument("site_id", siteId)
+				.WithArgument("calling_type", callingType)
+				.WithArgument("status", status)
+				.WithArgument("department", department)
+				.WithArgument("cost_center", costCenter)
+				.WithArgument("keyword", keyword)
+				.WithCancellationToken(cancellationToken)
+				.AsObject<PhoneCallUserProfilesPaginationObject>();
```

Comment:
If it turns out that you can indeed use `PaginatedResponseWithToken<T>`, as I suspect, you'll be able to use the following:
```suggestion
				.PaginatedResponseWithToken<PhoneCallUserProfile>("users");
```

Reply:
Yes, thanks! I didn't know that I could specify json property name for PaginatedResponseWithToken' `Records` property by just passing the name in arguments into `.AsPaginatedResponseWithToken<PhoneCallUserProfile>("users");`. Great solution! 
I've changed it as you suggested and removed unnecessary `PhoneCallUserProfilesPaginationObject` model.